src/styles/index.scss

```diff
@@ -1,3 +1,132 @@
+$star-active: url("/images/star-active.svg");
+$star: url("/images/star.svg");
+$bg-image: url(../images/imac.jpeg);
+$main-accent: #060b35;
+$secondary: #616070;
+$white: #fff;
+$blue-accent: #00acdc;
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  width: 198px;
+  height: 406px;
+  justify-content: space-between;
+  align-items: center;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__monoblock-img {
+    background-image: $bg-image;
+    background-size: cover;
+    width: 160px;
+    height: 134px;
+    margin: 32px 16px 40px;
+    display: block;
+  }
+
+  &__title-product {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0 16px 4px;
+    padding: 0;
+    color: $main-accent;
+  }
+
+  &__product-code {
+    color: $secondary;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin: 0 84px 16px 16px;
+  }
+
+  &__cost {
+    display: flex;
+    justify-content: space-between;
+    margin: 24px 16px 16px;
+    width: 166px;
+
+    &-price {
+      font-size: 12px;
+      line-height: 18px;
+      color: $secondary;
+    }
+
+    &-value {
+      color: $main-accent;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__buy {
+    background-color: $blue-accent;
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    margin: 0 16px 16px;
+
+    &:hover {
+      background: $white;
+      border: 1px solid $blue-accent;
+    }
+
+    &-link {
+      text-transform: uppercase;
+      text-decoration: none;
+      font-size: 14px;
+      font-weight: 700;
+      line-height: 16px;
+      color: $white;
+      width: 100%;
+      height: 100%;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+
+      &:hover {
+        color: $blue-accent;
+      }
+    }
+  }
+}
+
+.stars {
+  display: flex;
+  width: 166px;
+  white-space: nowrap;
+  align-items: center;
+
+  &__star {
+    background-image: $star;
+    height: 16px;
+    width: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
+
+  &__reviews {
```

Comment:
shouldn't be part of `stars` block. You have `stars` block from previous task - you should just reuse it